lib/fpmon.gi

```diff
@@ -210,7 +210,7 @@ function( F, rels )
     if Length(gens) > Length(rels) then
       SetIsFinite(s, false);
     fi;
-	
+        
```

Comment:
Is there any actual changes up to this point in this file? Looks like fiddling with whitespace.